CHANGELOG.rst

```diff
@@ -1,45 +1,53 @@
 Change Log
-==========
+##########
```

Comment:
I noticed ">>>>>>> master" in this file when reviewing. Would you mind cleaning that up?